apps/ccf-rui/src/app/shared/components/rotation-slider/rotation-slider.component.ts

```diff
@@ -66,4 +82,23 @@ export class RotationSliderComponent {
     this.ga.event('rotation_reset', 'rotation_slider');
     this.rotationChange.emit(this.rotation);
   }
+
+  displaySlider(dimension: 'x' | 'y' | 'z'): void {
+    this.displayedSlider = dimension;
+  }
+
+  @HostListener('document:keydown', ['$event'])
```

Comment:
Use the host property in the component decorator instead of `HostListener`. https://angular.dev/api/core/Component#host